src/modules/git_commit.rs

```diff
@@ -40,6 +40,39 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         )),
     );
 
+    if !config.tag_disabled {
+        // Let's get repo tags names
+        let tag_names = git_repo.tag_names(None).unwrap();
+
+        let tag_and_refs = tag_names.iter().flatten().flat_map(|name| {
+            let full_tag = format!("refs/tags/{}", name);
+            git_repo
+                .find_reference(&full_tag)
+                .map(|reference| (name, reference))
+        });
+
+        // Let's get HEAD commit id
+        let git_head_ref = git_repo.refname_to_id("HEAD").unwrap();
```

Comment:
Similarly for the `unwrap()` here. It would be best if we handled these without resorting to `unwrap()`.